src/main/kotlin/com/cosmotech/api/azure/eventhubs/AzureEventHubsClient.kt

```diff
@@ -133,6 +135,8 @@ class AzureEventHubsClient(private val csmPlatformProperties: CsmPlatformPropert
         "${scenarioRunMetaData.simulationRun},${scenarioRunMetaData.scenarioId}," +
             "${scenarioRunMetaData.scenarioRunStartTime}"
     val eventData = EventData(data)
+    val dropTag = "drop-by:" + scenarioRunMetaData.scenarioId
```

Comment:
the drop-by key here should be the csmSimulationId (scenarioRunMetaData.simulationRun ?) to be able to delete 1 scenario run and have the related data deleted: probesmeasures, simulationtotalfacts, scenariorunmetadata.
In current implementation all scenarunrunmetadata data will be be deleted for the scenario and not the data for only 1 scenario run.

Reply:
In the case that there no simulationrun, when the user delete a scenario we have to delete all scenario metadata that why the drop-by key has to be 'scenarioId'

Reply:
I do not agree: 
* No scenario run = no need to delete from scenariorunmetadata
* csmSimulationRun is a SimulationRun property
* scenariometadata should be deleted in the ScenarioServiceImpl.deleteScenario endpoint not scenariorun